Dockerfile

```diff
@@ -66,35 +91,21 @@ RUN apt-get remove --quiet --quiet $INSTALL_DEPS
 RUN apt-get --quiet --quiet clean
 RUN rm -rf /var/lib/apt/lists/*
 
-
-###
-# Setup the user and its home directory
-###
-FROM install AS setup_user
-
 ###
-# Create unprivileged user
+# Setup working directory and entrypoint
 ###
-RUN groupadd -r $USER
-RUN useradd -r -c "$USER user" -g $USER $USER
 
 # Put this just before we change users because the copy (and every
 # step after it) will always be rerun by docker, but we need to be
 # root for the chown command.
-COPY . $HOME
-RUN chown -R ${USER}:${USER} $HOME
-
-
-###
-# Setup working directory and entrypoint
-###
-FROM setup_user AS final
+COPY src ${CISA_HOME}
+RUN chown -R ${CISA_USER}:${CISA_GROUP} ${CISA_HOME}
 
 ###
 # Prepare to Run
 ###
 # Right now we need to be root at runtime in order to create files in
-# /home/shared
-# USER ${USER}:${USER}
-WORKDIR $HOME
+# ${CISA_HOME}/shared
+# USER ${CISA_USER}:${CISA_GROUP}
```

Comment:
Would you test access with my UID default change suggested above? That is what I needed to do to allow my Docker images to correctly read/write `cyhy` user (the user in the environment where this is deployed) files as an unprivileged user.

Reply:
This is because the `cyhy` user UID is explicitly set to `2048` [here](https://github.com/cisagov/cyhy_amis/blob/a0a1c08ccdee069085653841e169937940579358/terraform/cloud-init/cyhy_user_ssh_setup.yml#L9), correct?

Reply:
Correct. That should give the unprivileged user the same UID to access any `cyhy`-only files/directories.

Reply:
Please see commit c64baf1.

Reply:
On second thought, I reverted this change in commit e79adf1.  For this to work, I would need _all_ the Docker containers from [cisagov/orchestrator](https://github.com/cisagov/orchestrator) to do this.  This is because the containers write files to the shared directory that other containers read.

I will, however, make this change as a set of new PRs once this PR and cisagov/saver#61 are both merged.

Reply:
Will you make an appropriate issue in [cisagov/orchestrator](https://github.com/cisagov/orchestrator) and a comment referencing it to both of these projects so we don't lose track of that change?

Reply:
Please see cisagov/orchestrator#130.